ui/src/tasks/components/TaskRunsPage.tsx

```diff
@@ -31,13 +45,29 @@ class TaskRunsPage extends PureComponent<Props & WithRouterProps> {
           </Page.Header>
           <Page.Contents fullWidth={false} scrollable={true}>
             <div className="col-xs-12">
-              <TaskRunsList taskID={params.id} runs={dummyData} />
+              <TaskRunsList taskID={params.id} runs={runs} />
             </div>
           </Page.Contents>
         </Page>
-      </>
+      </SpinnerContainer>
     )
   }
+
+  public async componentDidMount() {
```

Comment:
Since this isn't setting state anymore, this can be
```
  public componentDidMount() {
    this.props.getRuns(this.props.params.id)
  }
```
I think that makes it a little more clear that it is issuing a side effect that takes place outside of the component.

Reply:
Okay, I removed it